prepare-commit-msg

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# WordPress prepare-commit-msg hook
+
+DEV_LIB_COMMIT_MESSAGE_FILE="/tmp/.dev-lib-commit-message.$PPID"
+COMMIT_EDITMSG=$1
+
+JIRA_TASK_REGEXP='.*([A-Z]{1,32}-[0-9]{1,32}).*'
+JIRA_BRANCH_REGEXP='([a-zA-Z]+-[0-9]{1,32})';
```

Comment:
So match issues that appear like `ABC-123-awesome-feature` or `feature/ABC-123-awesome`.